.eslintrc

```diff
@@ -37,6 +37,10 @@
     "no-multiple-empty-lines": ["error", { "max": 1 }],
     "jsx-quotes": ["error", "prefer-double"],
     "array-bracket-spacing": ["error", "never"],
+    "lines-around-comment": ["error", {
+      "allowBlockStart": true
+    }],
+    "parameter-properties": "off", // Disabled in favor of @typescript-eslint/parameter-properties
```

Comment:
Does this rule even exist? Can we just remove it?

Reply:
You're right, good catch!